From 3ae3bd4fd43442b7b03fdd07db53d4848bc2da93 Mon Sep 17 00:00:00 2001
From: Harshit Sharma <hsharma@cse.iitk.ac.in>
Date: Sun, 4 Mar 2018 09:14:22 +0530
Subject: [PATCH 3/3] changes done hello.c

---
 hello.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/hello.c b/hello.c
index 9a8af0a..d8954f2 100644
--- a/hello.c
+++ b/hello.c
@@ -1,6 +1,8 @@
 #include<stdio.h>
  void microkernel_sendmsg(char *);
-
+ void bro(){
+  printf("change is made");
+}
  void main(){
  printf("Helloworld!\n");
  printf("This must be a monolithic design\n");
-- 
2.7.4

